src/Resources/config/services.yml

```diff
@@ -0,0 +1,8 @@
+services:
+    ContaoCommunityAlliance\Translator\Contao\ContaoTranslatorService:
+        arguments:
+            - '@event_dispatcher'
+
+    contao.translator:
```

Comment:
We should use our own vendor here. My suggestion for the service name `cca.translator.contao_translator`. Maybe we should discuss some naming conventions in general.